dymos/examples/oscillator/doc/test_doc_oscillator.py

```diff
@@ -422,7 +422,7 @@ def test_ivp_driver_run_problem(self):
 
         # Now we're using the optimization driver to iteratively run the model and vary the
         # phase duration until the final y value is 0.
-        dm.run_problem(prob, refine=False, refine_iteration_limit=10)
+        dm.run_problem(prob, refine_iteration_limit=10, run_driver=False)
```

Comment:
We can remove `refine_iteration_limit=10` and `run_driver=False` here.  The previous version didn't refine but did run the driver.